ui/pages/token-details/token-details-page.js

```diff
@@ -0,0 +1,229 @@
+    nickname: state.metamask.provider.nickname,
+    type: state.metamask.provider.type,
+  }));
+
+  const { nickname: networkNickname, type: networkType } = currentNetwork;
+
+  const [copied, handleCopy] = useCopyToClipboard();
+
+  const renderTokenDetailsComponent = () => {
+    return (
+      <Box className="page-container token-details">
+        <Box marginLeft={5} marginRight={6}>
+          <Typography
+            fontWeight={FONT_WEIGHT.BOLD}
+            margin={[4, 0, 0, 0]}
+            variant={TYPOGRAPHY.H6}
+            color={COLORS.BLACK}
+            className="token-details__title"
+            data-testid="token-details-page__title"
+          >
+            {t('tokenDetails')}
+            <button
+              data-testid="token-details-page__close-button"
+              onClick={() => history.push(`${ASSET_ROUTE}/${token.address}`)}
+              className="token-details__closeButton"
+            />
+          </Typography>
+          <Box display={DISPLAY.FLEX} marginTop={4}>
+            <Typography
+              align={TEXT_ALIGN.CENTER}
+              fontWeight={FONT_WEIGHT.BOLD}
+              margin={[0, 5, 0, 0]}
+              variant={TYPOGRAPHY.H4}
+              color={COLORS.BLACK}
+              className="token-details__token-value"
+              data-testid="token-details-page__token-value"
+            >
+              {tokenBalance}
+            </Typography>
+            <Box marginTop={1}>
+              <Identicon
+                diameter={32}
+                address={token.address}
+                image={tokenMetadata ? imagePath : token.image}
+              />
+            </Box>
+          </Box>
+          <Typography
+            margin={[4, 0, 0, 0]}
+            variant={TYPOGRAPHY.H7}
+            color={COLORS.UI4}
+            data-testid="token-details-page__token-currency-balance"
+          >
+            {tokenCurrencyBalance || ''}
+          </Typography>
+          <Typography
+            margin={[6, 0, 0, 0]}
+            variant={TYPOGRAPHY.H9}
+            color={COLORS.UI4}
+            fontWeight={FONT_WEIGHT.BOLD}
+            data-testid="token-details-page__token-contract-address-title"
+          >
+            {t('tokenContractAddress')}
+          </Typography>
+          <Box display={DISPLAY.FLEX}>
+            <Typography
+              variant={TYPOGRAPHY.H7}
+              margin={[2, 0, 0, 0]}
+              color={COLORS.BLACK}
+              overflowWrap={OVERFLOW_WRAP.BREAK_WORD}
+              className="token-details__token-address"
+              data-testid="token-details-page__token-contract-address"
+            >
+              {token.address}
+            </Typography>
+            <Tooltip
+              position="bottom"
+              title={copied ? t('copiedExclamation') : t('copyToClipboard')}
+              containerClassName="token-details__copy-icon"
+            >
+              <button
```

Comment:
[REQ] you are using a native button element versus our Button component here, please fix.

Reply:
@brad-decker 
Fixed in: fc353df083ca4099424224735ae80831198a6dcd